src/pdf.ts

```diff
@@ -91,6 +91,24 @@ class PDFJSViewer extends Widget implements DocumentRegistry.IReadyWidget {
     return this._ready.promise;
   }
 
+  /**
+   * Get the scroll position.
+   */
+  getScroll(): PDFJSViewer.IPosition {
+    return {
+      page: this._pdfViewer.currentPageNumber,
+      x: 0,
```

Comment:
Is this where you would need to update it to pass information about the x and y values?

Reply:
Yep.